ColabTool/Shared/PostHeader/story.jsx

```diff
@@ -37,7 +37,7 @@ const noUserNameViaApiDraftDetails = {
   avatarUrl: 'https://buffer-uploads.s3.amazonaws.com/510521020a19000b6a00001e/a476fed03b1de4e06563d6063d7d3ee0.jpg',
   email: 'ash@buffer.com',
   via: 'api',
-  createdAt: 'March 2nd at 12:45pm (GMT)',
+  createdAt: 'on March 2nd at 12:45pm (GMT)',
```

Comment:
I don't think we need to add anything here to the `createdAt`  string since will be coming from buffer-web.